docs/usage.rst

```diff
@@ -0,0 +1,34 @@
+Twistd ServiceMaker Plugin Usage
+================================
+
+This plugin allows manual configuration of the HTTP Proxy.
+
+First, build this locally::
+
+	$ virtualenv ve
+	$ source ve/bin/activate
+	(ve)$ pip install -e .
+
+To see list of all twistd plugins::
+
+	(ve)$ twistd -d
+
+In order to configure the proxy using default settings, a blacklist must be specified as follows::
+
+	(ve)$ twistd -n vumi_http_proxy --blacklist=BLACKLISTFILENAME
+
+This blacklist must be a PyYAML configuration file - an example of which can 
+be found in ./docs/proxy_blacklist.yml
```

Comment:
If you like you can include the example blacklist using a literalinclude:
http://www.sphinx-doc.org/en/stable/markup/code.html#directive-literalinclude